examples/server.js

```diff
@@ -10,11 +10,11 @@ function listDirs(root) {
   var files = fs.readdirSync(root);
   var dirs = [];
 
-  for (var i=0, l=files.length; i<l; i++) {
+  for (var i=0, l=files.length; i<l; i++){
     var file = files[i];
-    if (file[0] !== '.') {
+    if (file[0] !== '.'){
```

Comment:
format: style mismatch:
```suggestion
    if(file[0] !== '.') {
```

`8418e2cd` ␣ at column 7 should be removed.
`39615007` ␣ should be inserted at column 25.